app/models/manageiq/providers/ovirt/inventory/collector.rb

```diff
@@ -1,8 +1,5 @@
 class ManageIQ::Providers::Ovirt::Inventory::Collector < ManageIQ::Providers::Inventory::Collector
   # TODO: review the changes here and find common parts with ManageIQ::Providers::Ovirt::InfraManager::Inventory::Strategies::V4
```

Comment:
This comment was attached to those require lines... Not sure if it should move somewhere else

Reply:
We have similar/copied lines here: https://github.com/ManageIQ/manageiq-providers-red_hat_virtualization/pull/40/files#diff-f2f5d4e887f15b36afacf467c592be51b1b1055f4aa04d5c553f77042620c671R2

Not sure what it's referring to but I can come back to it when I finish up the other PRs.  Maybe it was prior code that was removed and left orphaned. https://twitter.com/nzkoz/status/538892801941848064